__tests__/app.test.js

```diff
@@ -158,16 +158,87 @@ describe('POST /api/articles/:article_id/comments', () => {
                 const { comment } = body;
                 expect(comment.body).toBe('This is a test comment');
                 expect(comment.author).toBe('butter_bridge')
+                expect(comment.votes).toBe(0)
+                expect(comment.article_id).toBe(1)
+                
             });
     
     });
-    test('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
+    test('GET:400 sends an appropriate status and error message when given a valid but non-existent id', () => {
         return request(app)
           .post('/api/articles/999/comments')
           .expect(400)
           .then((response) => {
             expect(response.body.message).toBe('Bad request');
           });
       });
-    
 });
+
+describe('PATCH /api/articles/:articles_id', () =>{
+    test('PATCH 200: change vote count by increment for article by article_id', () =>{
+        const newVote = {
+            inc_votes: 100,
+            article_id: 1
+        }
+        return request(app)
+        .patch(`/api/articles/${newVote.article_id}`)
+        .send(newVote)
+        .expect(200)
+        .then((response) =>{
+            const {vote} = response.body;
+            expect(vote).toBe(200)
```

Comment:
It might be nice to test for the entire response from this endpoint, currently we could pass this test by only sending back this one property when we do want the whole response.